### PR TITLE
Implemented event certainty module

### PR DESCRIPTION
**Pull request by Mloweedgar**
ref https://github.com/CodeTanzania/ewea/issues/16

### CODE REVIEW

File: src/Events/components/EventCertainty/Form/index.js [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+} from '@codetanzania/ewea-api-states';
+import { Button, Form, Input } from 'antd';
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import { notifyError, notifySuccess } from '../../../../util';
+
+/**
+ * @class
+ * @name EventCertaintyForm
+ * @description  Render form for creating a new event group
```
Comment (rijkerd) [inline src/Events/components/EventCertainty/Form/index.js]: Change to event certainty

File: src/Events/components/EventCertainty/index.js [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+} from '@codetanzania/ewea-api-states';
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import { Modal } from 'antd';
+import Topbar from '../../../components/Topbar';
+import EventCertaintyForm from './Form';
+import EventCertaintiesList from './List';
+import './styles.css';
+
+/* constants */
```
Comment (rijkerd) [inline src/Events/components/EventCertainty/index.js]: Remove this line

**Review by rijkerd — CHANGES_REQUESTED** [review]

**Comment by BenMaruchu** [comment]
@Mloweedgar can you also resolve conflicts on this branch

**Comment by Mloweedgar** [comment]
@BenMaruchu conflicts have been resolved

**Review by BenMaruchu — APPROVED** [review]

**Pull request merged**